marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/customStep/getCustomStep.sjs

```diff
@@ -20,7 +20,7 @@ xdmp.securityAssert("http://marklogic.com/data-hub/privileges/read-custom", "exe
 const Artifacts = require('/data-hub/5/artifacts/core.sjs')
 var stepName;
 
-let dataHubOptions = ["sourceQuery", "sourceQueryIsScript", "constrainSourceQueryToJob", "provenanceGranularityLevel", "acceptsBatch",
+let dataHubOptions = ["sourceQuery", "sourceQueryLimit", "sourceQueryIsScript", "constrainSourceQueryToJob", "provenanceGranularityLevel", "acceptsBatch",
```

Comment:
What is the impact if someone forgets to register a new property here? This is a maintenance burden because any time we add a new option, someone has to remember that it goes here.